nncf/quantization/algorithms/fast_bias_correction/torch_backend.py

```diff
@@ -0,0 +1,144 @@
+            target_type == TargetType.POST_LAYER_OPERATION:
+            port_id = None
+        if target_type in PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP:
+            target_type = PTFastBiasCorrectionAlgoBackend.TARGET_TYPE_TO_PT_INS_TYPE_MAP[target_type]
+        return PTTargetPoint(target_type, target_node_name, input_port_id=port_id)
+
+    @staticmethod
+    def create_bias_correction_command(node: NNCFNode,
+                                       bias_value: np.ndarray,
+                                       nncf_graph: NNCFGraph) -> PTBiasCorrectionCommand:
+        return create_bias_correction_command(node, bias_value)
+
+    @staticmethod
+    def model_extraction_command(inputs: List[str], outputs: List[str]) -> PTModelExtractionCommand:
+        return PTModelExtractionCommand(inputs[0])
+
+    @staticmethod
+    def mean_statistic_collector(reduction_shape: ReductionShape,
+                                 num_samples: Optional[int] = None,
+                                 window_size: Optional[int] = None) -> PTMeanStatisticCollector:
+        return PTMeanStatisticCollector(reduction_shape, num_samples, window_size)
+
+    @staticmethod
+    def get_sub_input_output_names(subgraph: nn.Module) -> Tuple[str, str]:
+        # Pytorch does not have name for extracted node
+        return None, None
+
+    @staticmethod
+    def create_blob(shape: Tuple[int], data: List[np.ndarray], channel_axis: int) -> np.ndarray:
+        blob = torch.zeros(shape, dtype=data[0].dtype)
+        for j, idx in enumerate(np.ndindex(blob.shape[channel_axis])):
+            index = tuple(slice(None) if i != channel_axis else idx for i in range(blob.ndim))
+            blob[index] = data[j]
+        return blob
+
+    @staticmethod
+    def get_bias_value(node: NNCFNode, nncf_graph: NNCFGraph, model: nn.Module) -> np.ndarray:
+        node_module = model.get_containing_module(node.node_name)
+        if node_module.bias is None:
+            # Initialize bias as zeros for node node without bias
+            node_module.bias = nn.Parameter(
+                torch.zeros(
+                    node_module.weight.data.shape[0],
+                    dtype=node_module.weight.data.dtype,
+                    device=node_module.weight.data.device,
+                )
+            )
+        return node_module.bias.data
+
+    @staticmethod
+    def get_activation_port_ids_for_bias_node(node: NNCFNode) -> Tuple[int, int]:
+        return 0, 0
+
+    @staticmethod
+    def process_model_output(raw_data: Dict, output_name: str) -> PTNNCFTensor:
+        return PTNNCFTensor(raw_data)
+
+    @staticmethod
+    def is_quantized_weights(node: NNCFNode, nncf_graph: NNCFGraph, model: nn.Module) -> bool:
+        node_module = model.get_containing_module(node.node_name)
+        for pre_op in node_module.pre_ops.values():
+            if isinstance(pre_op.op, (BaseQuantizer, FakeQuantize)):
```

Comment:
What happens in the case of only activation input quantization?

Reply:
Nothing, return false. It will mean that node (Conv, Matmul) will not quantize.
@vshampor may be you know another way to check quantized node?

Reply:
Based on the code, this method will return `True`, because there is no check for the `weights` value. Checking based only on the `isinstance` of one of the inputs is wrong since we need to check the quantized **weights** not **activations or weights**.

Reply:
Add check that FQ for weight